web/src/utils/convert.ts

```diff
@@ -0,0 +1,21 @@
+export const uuidToURL = (uuid: string, filename: string) => {
+  // TODO: needs proper implementation
+  const host = window.REEARTH_CONFIG?.host ?? "http://localhost:8080";
+  const path = getPathFromUUID(uuid, filename);
+  return join(host, path);
+};
+
+export const getPathFromUUID = (uuid: string, filename: string) => {
+  if (!uuid || !filename) {
```

Comment:
Your types for uuidToURL and getPathFromUUID don't allow for either param to be undefined or null. If you want to allow that, update the param types. If not, you don't need this if statement

Reply:
This if statement also checks if the string is empty.